src/scrape_up/timeanddate/day_in_history.py

```diff
@@ -0,0 +1,206 @@
+                            Act of July 1962, which was a result of anti-colonial sentiments that were spreading
+                            throughout the globe, gave Jamaica full independence by leaving the Federation of the 
+                            West Indies.",
+            "1945 US Bombs Hiroshima": "In the first of the only two times nuclear weapons have been used in warfare, 
+                            the United States dropped a nuclear bomb, nicknamed Little Boy on the industrial city
+                            of Hiroshima in Japan. Over 150,000 people were estimated killed by the resulting 
+                            explosion. Japan had joined the Second World War in December 1941 on the side of the
+                            Axis powers. After the Hiroshima and Nagasaki bombings 3 days later on August 9,
+                            Japan surrendered to the Allies and ended the Pacific War.", 
+            "1926 First Woman to Swim Across the English Channel": "Gertrude Ederle, an American Olympic
+                            swimmer swam across the English channel  a body of water between England and
+                            France, in 14 hours and 34 minutes. Only 5 other people, all men, had swum 
+                            across the channel before Ederle. "
+        }
+        ```
+        """
+
+        try:
+
+            x = self.page_soup.find("ul", {"class":"list--big"})
+            x = x.get_text().split("\n")
+            x = x[1:-1]
+            history_dic = {}
+
+            for i in range(0, len(x), 3):
+                history_dic[x[i]] = x[i + 1]
+            
+            return history_dic
+
+        except:
+            return None
+    
+    def holidays(self):
+
+        """
+        Create an instance of `DayInHistory` class.\n
+
+        ```python
+        dayinhistory = DayInHistory()
+        dayinhistory.holidays()
+        ```
+
+        Return\n
+        ```js
+        {
+            "American Family Day": "USA",
+            "Celebrations of San Salvador": "El Salvador",
+            "Friendship Day": "India",
+            "Hiroshima Memorial Day": "Japan",
+            "Independence Day": "Bolivia, Jamaica"
+        }
+        ```
+        """
+
+        try:
+
+            x = self.page_soup.find("div", {"class" : "sidebar-holidays"})
+            x = x.get_text().split("\n")[5 : -1]
+            x = [[z.strip() for z in y.split("-")] for y in x]
+
+            return dict(zip([y[0] for y in x], [y[1] for y in x]))
+        
+        except:
+            return None
+
+
+    def births(self):
+
+        """
+        Create an instance of `DayInHistory` class.\n
+
+        ```python
+        dayinhistory = DayInHistory()
+        dayinhistory.births()
+        ```
+
+        Return\n
+        ```js
+        {
+            "1983 Robin van Persie": "Dutch footballer",
+            "1928 Andy Warhol": "American artist",
+            "1911 Lucille Ball": "American actress",
```

Comment:
The same goes for this method. Return list objects, where each object has keys year, and births.